lib/duffel_api/services/offers_service.rb

```diff
@@ -22,13 +22,19 @@ def list(options = {})
       end
 
       # Returns an `Enumerator` which can automatically cycle through multiple
-      # pages of `Resources;:Offers`s
+      # pages of `Resources;:Offers`s.
```

Comment:
Tiny nitpick on Ruby class name syntax.
```suggestion
      # pages of `Resources::Offers`s.
```

Reply:
Good spot! Bad copying and pasting on my part.